tests/analysis_tests/tests-reanalyze/deadcode/src/DeadTest.res

```diff
@@ -95,33 +95,10 @@ and bar = () => foo()
 
 let withDefaultValue = (~paramWithDefault=3, y) => paramWithDefault + y
 
-external unsafe_string1: (bytes, int, int) => Digest.t = "caml_md5_string"
```

Comment:
Here and other files: when there's stuff that does not exist anymore, or e.g. only one component is now supported per file, just removed the code.